app/scripts/modules/core/delivery/executions/executions.html

```diff
@@ -48,7 +48,7 @@ <h3 class="filters-placeholder">
             <option value="timeBoundary">Time Boundary</option>
           </select>
         </div>
-        <div class="form-group">
+        <div class="form-group" ng-if="vm.filter.groupBy === 'name'">
```

Comment:
Not true. It's used to determine how many executions to fetch per pipeline, which determines how many pipelines are actually shown on the screen.